docs/03-TechnicalInfrastructure.md

```diff
@@ -357,3 +379,97 @@ Any native scan inserted into the `files` table can be processed and the output
       to the `type` field of the `ImagingFileTypes` table.
   
 ![processed_tables](images/Processed_data_tables.png)
+
+
+### 3.2.2 Database infrastructure for the electrophysiology insertion pipeline
+
+The database infrastructure for the electrophysiology data is composed of 
+five main tables: 
+- `physiological_file` and `physiological_parameter_file`: contain information 
+specific to the electrophysiology file and a link to the `session` table
+- `physiological_channel`: contains information regarding the channels used 
+to record the electrophysiology data if any provided in the BIDS structure
+- `physiological_electrode`: contains information regarding the electrodes 
+used to record the electrophysiology data if any provided in the BIDS structure
+- `physiological_task_event`: contains information about the task events that 
+were used during the recording if any task was used in the paradigm
+
+![OverallPhysiologicalTables](images/OverallPhysiologicalTables.png)
+
+
+### 3.2.2.1 The `physiological_file` and `physiological_parameter_file` tables
+
+These two tables regroups information about the electrophysiology files. 
+Typically, the `physiology_file` will store one recording per row linking it 
+to the `session` table. This table is also linked to the following tables 
+using the `PhysiologicalFileID` key:
+- `physiological_modality`: storing information about the recording modality 
+(*e.g.* 'eeg', 'meg', 'ieeg'...)
+- `physiological_output_type`: storing information about the BIDS output type 
+(*e.g.* 'raw' or 'derivatives')
+- `ImagingFileType`: storing the file type of the electrophysiology recording
+ (*e.g.* 'set', 'bdf', 'cnt', 'edf', 'vhdr', 'vsm'...)
+- `physiological_parameter_file`: containing all the information stored in the 
+BIDS JSON sidecar file of the electrophysiology recording
+
+![PhysiologicalMainTables](images/PhysiologicalMainTables.png)
+
+
+### 3.2.2.2 The `physiological_channel` tables
+
+The `physiological_channel` table is populated using data from the 
+`*_channels.tsv` BIDS file when present with the electrophysiology recording. 
+If the `*_channels.tsv` file is not present in the BIDS dataset, then no 
+entry will be inserted in this table. This table is linked to the 
+`physiological_file` table via the `PhysiologicalFileID` foreign key. In 
+addition, `physiological_channel` is linked to the following tables:
+- `physiological_status_type`: this table contains two entries specifying the 
```

Comment:
{good, bad, null?} (can it have a null value?) 
Is status actually reflective of QC?

Reply:
this is a lookup table for the channel status. The channel table will allow NULL but only good and bad are the possible values when that data is set. I am not sure if it comes from an automatic check of whether the channel was good or preliminary QC. It is not always there actually.